src/js/server/devpmt-express.js

```diff
@@ -0,0 +1,320 @@
+/* eslint-env node */
+"use strict";
+
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+var JSON5 = require("json5");
+var path = require("path");
+fluid.require("gpii-express");
+fluid.require("gpii-handlebars");
+require("universal");
+
+fluid.registerNamespace("gpii.devpmt");
+fluid.registerNamespace("gpii.handlebars");
+
+fluid.module.register("gpii-devpmt", __dirname, require);
+
+/** gpii.handlebars.requestFuncTransform
+ *  This component is sort of a facsimile of the functions available
+ *  in many web frameworks that take a request instance or URL parameter
+ *  and return something for the response. In this case what is returned
+ *  will be assigned to the variable in the transform.
+ */
+fluid.defaults("gpii.handlebars.requestFuncTransform", {
+    gradeNames: ["fluid.standartInputTransformFunction"]
+});
+
+gpii.handlebars.requestFuncTransform = function (value, transformSpec) {
+    return fluid.invokeGlobalFunction(transformSpec.func, [transformSpec.that, value]);
+};
+
+/**
+ * gpii.devpmt.npset - Infusion component representing a single NP Set for
+ * a user or snapset. Based on current hardwire, loads it's NP Set from
+ * the bundled GPII's testData.
```

Comment:
Only it doesn't, it loads it from your copypaste of it...

Reply:
These are reading from universal now :p